GitUI/CommandsDialogs/AboutBox.cs

```diff
@@ -15,6 +15,7 @@ public AboutBox()
 
             InitializeComponent();
             Translate();
+            this.AdjustForDpiScaling();
```

Comment:
why not do this on the base form/base usercontrol?

Reply:
Can't think of a reason, so long as we test every form.

Should only be needed on the form, as it will find the user controls.